app/response_processor.py

```diff
@@ -146,64 +154,84 @@ def send_notification(self, survey_id):
                 self.logger.info("Ignoring received CTP submission")
             else:
                 self.logger.info("About to publish notification to queue")
-                self.notifications.publish_message(self.tx_id, headers={'tx_id': self.tx_id})
+                self.notifications.publish_message(
+                    self.tx_id, headers={
+                        'tx_id': self.tx_id
+                    })
         except PublishMessageError as e:
             self.logger.error("Unable to queue response notification", error=e)
             raise RetryableError
 
     def decrypt_survey(self, encrypted_survey):
         self.logger.info("Decrypting survey")
```

Comment:
I would have removed this log. Does not give any real information and polute the log. Or I would attach the survey decripted in the final logger.info . I know is not part of the code review, but make sense fix things whne you can

Reply:
The 'Decrypting survey" message is bound with the tx_id and ru_ref so comes in useful when tracing transactions through the system (particularly in Splunk).

Reply:
This log line is used in Splunk to alert is to queue errors.